src/Carbon/Lang/sk.php

```diff
@@ -33,33 +33,87 @@
  * - Marek Adamický
  * - AlterwebStudio
  */
+
+use Carbon\CarbonInterface;
+
+$fromNow = function ($time) {
+    return 'o '.strtr($time, [
+            'hodina' => 'hodinu',
+            'minúta' => 'minútu',
+            'sekunda' => 'sekundu',
+        ]);
+};
+
+$ago = function ($time) {
+    $replacements = [
+        '/(\b|$)hodina(\b|$)/' => 'hodinou',
+        '/(\b|$)minúta(\b|$)/' => 'minútou',
+        '/(\b|$)sekunda(\b|$)/' => 'sekundou',
+        '/(\b|$)deň(\b|$)/' => 'dňom',
+        '/(\b|$)týždeň(\b|$)/' => 'týždňom',
+        '/(\b|$)mesiac(\b|$)/' => 'mesiacom',
+        '/(\b|$)rok(\b|$)/' => 'rokom',
```

Comment:
```suggestion
        '/\bhodina\b/' => 'hodinou',
        '/\bminúta\b/' => 'minútou',
        '/\bsekunda\b/' => 'sekundou',
        '/\bdeň\b/' => 'dňom',
        '/\btýždeň\b/' => 'týždňom',
        '/\bmesiac\b/' => 'mesiacom',
        '/\brok\b/' => 'rokom',
```
`\b` includes `$` and `$` cannot be at the beginning